client/src/common/constants/enums.js

```diff
@@ -65,3 +65,15 @@ export const PasswordValidationValues = Object.freeze({
   MIN: 4,
   MAX: 32
 });
+
+export const NotificationFrequency = Object.freeze({
+  FRIDAY: 'notification/FRIDAY',
+  MONDAY: 'notification/MONDAY',
+  NEVER: 'notification/NEVER',
+  SATURDAY: 'notification/SATURDAY',
+  SUNDAY: 'notification/SUNDAY',
+  THURSDAY: 'notification/THURSDAY',
+  TUESDAY: 'notification/TUESDAY',
+  WEDNESDAY: 'notification/WEDNESDAY',
+  WEEKLY: 'notifications/WEEKLY'
```

Comment:
why weekly is here? Is it needed?

Reply:
yes it is needed to recognize that someone chose weekly and DAYS select needs to be visible.

It is not saved in the DB. it is only for select days visibility